data/FilterList.json

```diff
@@ -21146,5 +21146,267 @@
+    "licenseId": 35,
+    "name": "noads.online additional app specific rules",
+    "syntaxId": 3,
+    "viewUrl": "https://raw.githubusercontent.com/deletescape/noads/master/lists/additional-rules.txt"
+  },
+  {
+    "id": 2063,
+    "homeUrl": "https://nixnet.services/dns/",
+    "licenseId": 2,
+    "name": "hBlock Hosts Mini",
+    "syntaxId": 1,
+    "viewUrl": "https://nixnet.services/hosts.txt"
+  },
+  {
+    "id": 2064,
+    "homeUrl": "https://nixnet.services/dns/",
+    "licenseId": 24,
+    "name": "hBlock Hosts Mini (DNS)",
+    "syntaxId": 22,
+    "viewUrl": "https://adblock.any.dns.nixnet.xyz/dns-query",
+    "viewUrlMirror1": "tls://198.251.90.89:853",
+    "viewUrlMirror2": "198.251.90.89"
+  },
+  {
+    "id": 2065,
+    "homeUrl": "https://github.com/troysjanda/MyBlockLists/",
+    "issuesUrl": "https://github.com/troysjanda/MyBlockLists/issues",
+    "name": "Xpi Yahoo Complete Filters",
+    "syntaxId": 4,
+    "viewUrl": "https://raw.githubusercontent.com/troysjanda/MyBlockLists/master/yahoocomplete.txt"
+  },
+  {
+    "id": 2066,
+    "homeUrl": "https://nixnet.services/dns/",
+    "licenseId": 24,
+    "name": "hBlock Hosts Mini (DNS)",
+    "syntaxId": 22,
+    "viewUrl": "https://adblock.any.dns.nixnet.xyz/dns-query",
+    "viewUrlMirror1": "tls://198.251.90.89:853",
+    "viewUrlMirror2": "198.251.90.89"
+  },
+  {
+    "id": 2067,
+    "description": "SecureDNS can be used as an Adblocker by using a alternative hostname/address instead. Links that didn't fit in the Subscribe button URL texts, include [2a03:b0c0:0:1010::e9a:3001]:5443",
+    "homeUrl": "https://securedns.eu/#ads",
+    "name": "notracking - DNS",
+    "syntaxId": 22,
+    "viewUrl": "https://ads-doh.securedns.eu/dns-query",
+    "viewUrlMirror1": "tls://ads-dot.securedns.eu",
+    "viewUrlMirror2": "146.185.167.43:5443"
+  },
+  {
+    "id": 2068,
+    "homeUrl": "https://www.powershellgallery.com/packages/Manage-AdBlocksOnDNSServer/",
+    "issuesUrl": "https://github.com/hechz/Powershell/issues",
+    "licenseId": 24,
+    "name": "Peter Lowe's List (PowerShell PKG)",
+    "syntaxId": 44
+  },
+  {
+    "id": 2069,
+    "description": "Peter Lowe's (as of August 2018) colleague Vladimir Pissarev (who is part of the Security team in his company) mentioned to him that he runs a public DNS server that not only blocks ads and trackers, but also malware and viruses as well.",
+    "homeUrl": "https://itsecurity.ee/",
+    "name": "Vladimir Pissarev's DNS Server",
+    "syntaxId": 22,
+    "viewUrl": "91.239.26.140"
+  },
+  {
+    "id": 2070,
+    "description": "I wanted an easy solution to blocking ads and tracking on all my devices and AdGuard DNS didn’t block quite enough for me, so I decided to run my own AdGuard Home server. Sharing is caring, which is why I’ve decided to make my server available for everyone! Links that didn't fit in the Subscribe button URL texts, include 2a03:b0c0:3:d0::5f18:6001"
```

Comment:
I think missing a comma at the end of this line.

Reply:
The syntax checker in "Files changed" seemed to point out the same problem as well, so I've tried to fix it now.